Makefile

```diff
@@ -423,9 +424,25 @@ all-openshift-tests = $(foreach image,$(all-openshift-tests-list),openshift-test
 .PHONY: openshift-tests
 openshift-tests: $(all-openshift-tests)
 
+# Define list of all tests
+all-controller-openshift-tests-list:=	features-openshift \
+														node \
+														drupal \
+														drupal-postgres \
```

Comment:
```suggestion
```
@shreddedbacon - suggest removing this line for now, Lagoon 1.x doesn't handle the renamed tests well yet